edgexfoundry/edgex-examples
# fix: Update remote device service of SSH for Jakarta release

Fixes https://github.com/edgexfoundry/edgex-docs/issues/515

Signed-off-by: Bryon Nevis <bryon.nevis@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:


## Testing Instructions
See https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/security/Ch-SSH-Tunneling-HowToSecureDeviceServices.md (doc update in progress)
